### PR TITLE
added taar_update_whitelist to the taar_amodump DAG

### PR DESCRIPTION
**Pull request by crankycoder**
Adds a job to extract the AMO editorial content and generate a JSON blob with all editorial reviewed GUIDs in an S3 bucket.

### CODE REVIEW

**Comment by crankycoder** [comment]
@acmiyaguchi  This has been in queue for ages - can we get this reviewed?

File: dags/taar_amodump.py [hunk 1]
```diff
@@ -48,6 +48,21 @@
...
+editorial_whitelist = EMRSparkOperator(
+    task_id="taar_update_whitelist",
+    job_name="Generate a JSON blob from editorial reviewed addons for TAAR",
+    execution_timeout=timedelta(hours=1),
+    instance_count=1,
+    owner="vng@mozilla.com",
+    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
+    env=mozetl_envvar("taar_update_whitelist",
+                      {},
+                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
```
Comment (acmiyaguchi) [inline dags/taar_amodump.py]: `MOZETL_SUBMISSION_METHOD` should be set to `spark` by default, and will be ignored by `MozDatabricksRunSubmit` if it's every moved over.

File: dags/taar_amodump.py [hunk 2]
```diff
@@ -48,6 +48,21 @@
...
 
+editorial_whitelist = EMRSparkOperator(
+    task_id="taar_update_whitelist",
+    job_name="Generate a JSON blob from editorial reviewed addons for TAAR",
+    execution_timeout=timedelta(hours=1),
+    instance_count=1,
+    owner="vng@mozilla.com",
+    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
+    env=mozetl_envvar("taar_update_whitelist",
+                      {},
```
Comment (acmiyaguchi) [inline dags/taar_amodump.py]: This is missing a required `--date` option from here -- I'm not sure whether it should be `ds` or `ds_nodash` though from a cursory glance. 

https://github.com/mozilla/python_mozetl/blob/d684691258d01946f7b58d62aefd414d63419221/mozetl/taar/taar_update_whitelist.py#L75

Comment (crankycoder) [reply dags/taar_amodump.py]: I've added the date in 92bf188

**Pull request merged**